documentation/modules/cruise-control/proc-deploying-cruise-control.adoc

```diff
@@ -32,23 +32,27 @@ spec:
   # ...
   cruiseControl:
     replicas: 1 <1>
-    config: <2>
+    capacity: <2>
+      nwIn: 10000
+      nwOut: 10000
+      # ...
+    config: <3>
       default.goals: >
          com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,
          com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal
       cpu.balance.threshold: 1.1
       metadata.max.age.ms: 300000
       send.buffer.bytes: 131072
       # ...
-    resources: <3>
+    resources: <4>
       requests:
         cpu: 200m
         memory: 64Mi
       limits:
```

Comment:
Again we should label limits in the docs and deal with them in the code

Reply:
I should probably make the distinction between CC application resource limits, Kafka resource limits, and CC capacity limits and what they mean for each other, I can add a comment in the docs similar to the one above if that makes things any more clear (unless I am misunderstanding this!). Let me know what you think